### ISSUE TITLE
Echoing? 

### ISSUE THREAD

**Issue by conklin75** [mchristopher/PokemonGo-DesktopMap#1244]
I'm having an issue where a pokemon will appear and I can go to get it and imediately after the original timer expires another of the same pokemon appears with a new 15ish minute timer but doesn't seem to exist. Not a huge problem but annoying. I've also noticed the issue also occurs on fastpokemap.se

**Comment by srenobdrawkwa** [comment]
Sounds like it's Probably just Doubled spawn times. So it's longer then 15 mins. Try with a shitty Pokemon and  If you don't catch the Pokemon Once the Spawn time expires it should still be there..

**Comment by ehsians** [comment]
That is interesting srenobdrawkwa, that might be true just to throw off PGO DesktopMaps.
Ran into several occasions when pokemons are captured, expired, and then reappears on the map.

**Comment by jasonasey** [comment]
the app doesnt register over 15 mins spawn times.

but its possible for it to be 15 mins 30 mins or an hour even.. So it just rescans the area and thinks it finds a new one when it is the same one.

**Comment by conklin75** [comment]
I think this is it I didn't know they could spawn for more than 15 minutes. Seems like it happens with bulbasaur and kakuna a lot nothing else really noticeable

**Comment by jasonasey** [comment]
it does happen with more rare pokemon as well, just not as often.

I've seen exeggutor and snorlax spawn for 30 mins and even 1 hour before.

**Comment by Burbankian** [comment]
they didn't echo, it just that there are many types of spawns  
1) the standard 15minutes
2) 30 minutes
3) 45 minutes
4) 60 minutes
5) 45 minutes spawn where the pokemon will be hidden between minute 16th-30th
6) 60 minutes spawn where the pokemon will be hidden between minute 16th-30th
7) 60 minutes spawn where the pokemon will be hidden between minute 31st-45th
8) 60minutes spawn where the pokemon will be hidden between minute 16th-45th  
the bottom line is, you can only get 1 pokemon per hour on a specific spawn point  
the map doesn't account spawn point type  
for more information on spawn points:
https://www.reddit.com/r/pokemongodev/comments/4yzqc2/spawn_point_types_clearing_confusion/
